src/cargo/ops/cargo_update.rs

```diff
@@ -410,41 +426,62 @@ pub fn write_manifest_upgrades(
                     dep_key_str,
                     dep_item,
                 )?;
+                let name = &dependency.name;
+
+                if let Some(renamed_to) = dependency.rename {
+                    trace!(
+                        "skipping dependency renamed from `{}` to `{}`",
+                        name,
+                        renamed_to
+                    );
+                    continue;
+                }
```

Comment:
nit: where it works, please prefer inline interpolation

```suggestion
                if let Some(renamed_to) = dependency.rename {
                    trace!(
                        "skipping dependency renamed from `{name}` to `{renamed_to}`"
                    );
                    continue;
                }
```
(ditto with `dependency.name` -> `name` changes)

Reply:
Fixed.